src/main/scala/org/broadinstitute/dsde/workbench/sam/dataAccess/PostgresAccessPolicyDAO.scala

```diff
@@ -569,6 +569,7 @@ class PostgresAccessPolicyDAO(
 
   override def deleteResource(resource: FullyQualifiedResourceId, leaveTombStone: Boolean, samRequestContext: SamRequestContext): IO[Unit] =
     serializableWriteTransaction("deleteResource", samRequestContext) { implicit session =>
+      deletePolicyMembersFromGroups(resource)
```

Comment:
is this still necessary here, given that `ResourceService` calls `removeSubjectFromPolicy`?

Reply:
I guess not!